webpack.development.js

```diff
@@ -10,6 +10,7 @@ const { mergeWithRules } = require('webpack-merge');
  * ForkTsCheckerWebpackPlugin provides type checking on top of webpack watch.
  */
 const common = {
+  devtool: false,
```

Comment:
fixes `unsafe-eval` errors in development, causing js to break

Reply:
haha I didn't notice this. just shipped an alternative fix: https://github.com/guardian/gateway/pull/837/commits/13599eac741b2be59e807fb10e1940be2dda17d6, letting the script eval only on dev envs. What do you think about that method? That way we can keep the source maps that I think devtool gives us.

Reply:
ah nice! thats a good approach, i'll also comment on that PR, but a slightly safer way of reading from `process.env` in gateway is to read from the `getConfiguration` method provided in `server/lib/configuration.ts`